controller/persistence/migration_initialize.go

```diff
@@ -338,7 +338,11 @@ var hostV1SchemaSansDefs = map[string]interface{}{
 				"properties": map[string]interface{}{
 					"connectTimeoutSeconds": map[string]interface{}{
 						"$ref":        "#/definitions/timeoutSeconds",
-						"description": "defaults to 5",
+						"description": "Timeout when making outbound connections. Defaults to 5. If both connectTimoutSeconds and connectTimeout are specified, connectTimeout will be used.",
```

Comment:
does the validator that we use recognize the [deprecated annotation](https://json-schema.org/understanding-json-schema/reference/generic.html)?

Reply:
Good idea, I'll give it a try :+1:

Reply:
done

Reply:
are you going to update the rendered json for host.v1?

Reply:
I did, it's got the deprecated marker in it now

Reply:
oh, I misread that as host.v2. I'll add the updated host.v1 as well.

Reply:
updated host.v1